docs/high-performance/Introduction.md

```diff
@@ -0,0 +1,52 @@
+keywords: windows 10, uwp, windows community toolkit, uwp community toolkit, uwp toolkit, get started, visual studio, high performance, net core, net standard
+---
+
+# Introduction to the High Performance package
+
+This package can be installed through NuGet, and it has the following multi-targets:
+- .NET Standard 1.4
+- .NET Standard 2.0
+- .NET Standard 2.1
+- .NET Core 2.1
+- .NET Core 3.1
+
+This means that you can use it from anything from UWP or legacy .NET Framework applications, games written in Unity, cross-platform mobile applications using Xamarin, to .NET Standard libraries and modern .NET Core 2.1 or .NET Core 3.1 applications. The API surface is almost identical in all cases, and lots of work has been put into backporting as many features as possible to older targets like .NET Standard 1.4 and .NET Standard 2.0. Except for some minor differences, you can expect the same APIs to be available on all target frameworks.
+
+The reason why multi-targeting has been used is to allow the package to leverage all the latest APIs on modern runtimes (like .NET Core 3.1) whenever possible, while still offering most of its functionalities to all target platforms. It also makes it possible for .NET Core 2.1 to use all the APIs that it has in common with .NET Standard 2.1, even though the runtime itself doesn't fully implement .NET Standard 2.1. All of this would not have been possible without multi-targeting to both .NET Standard as well as individual runtimes.
+
+Follow these steps to install the High Performance package:
+
+1. Open an existing project in Visual studio, targeting any of the following:
+    - UWP (>= 10.0)
+    - .NET Standard (>= 1.4)
+    - .NET Core (>= 1.0)
+    - Any other framework supporting .NET Standard 1.4 and up
+
+2. In Solution Explorer panel, right click on your project name and select **Manage NuGet Packages**. Search for **Microsoft.Toolkit.HighPerformance** and install it.
+
+    ![NuGet Packages](../resources/images/ManageNugetPackages.png "Manage NuGet Packages Image")
+
+3. Add a using directive in your C# files to use the new APIs:
+
+    ```c#
+    using Microsoft.Toolkit.HighPerformance;
+    ```
+
+4. If you want so see some code samples, you can either read through the other docs pages for the High Performance package, or have a look at the various [unit tests](https://github.com/windows-toolkit/WindowsCommunityToolkit/tree/master/UnitTests/UnitTests.HighPerformance.Shared) for the project.
+
+## When should I use this package?
+
+As the name suggests, the High Performance package contains a set of APIs that are heavily focused on optimization. All the new APIs have been carefully crafted to achieve the best possible performance when using them, either through reduced memory allocation, micro-optimizations at the assembly level, or by structuring the APIs in a way that facilitates writing performance oriented code in general.
+
+This package makes heavy use of APIs such as:
```

Comment:
Not sure if before or after, but we should have another part that calls out the other doc pages that you've written for the toolkit to highlight their scenarios (or the duality with the APIs here). As this is the introduction page. :)

Reply:
Not sure what you mean exactly, aren't the other docs pages already visible right below the introduction in the sidebar on the left? I tried out something that I hope is similar to what you had in mind in https://github.com/MicrosoftDocs/WindowsCommunityToolkitDocs/pull/301/commits/aafd2edb890e14aa4ae6e80efc117d7ce93723b4, let me know if that works or if there's anything else I should add! 😊